examples/prom/linear_elasticity_global_rom.cpp

```diff
@@ -0,0 +1,525 @@
+//               libROM MFEM Example: parametric ROM for linear elastic problem (adapted from ex2p.cpp)
+//
+// Compile with: ./scripts/compile.sh -m
+//
+// Description:  This example code demonstrates the use of MFEM and libROM to
+//               define a simple projection-based reduced order model of a 
+//				 simple linear elasticity problem describing a multi-material 
```

Comment:
This needs to be edited.

Reply:
I'll fix it!

Reply:
Just a question, what is it in particular that I should change about the description?

Reply:
The description is fine, it just needs the indenting aligned.